cvs diff: Diffing .
cvs diff: Diffing inc
cvs diff: Diffing prj
cvs diff: Diffing qa
cvs diff: Diffing qa/unoapi
cvs diff: Diffing sdi
cvs diff: Diffing source
Index: starmath/source/cfgitem.hxx
===================================================================
RCS file: /cvs/sw/starmath/source/cfgitem.hxx,v
retrieving revision 1.10
diff -u -r1.10 cfgitem.hxx
--- starmath/source/cfgitem.hxx	27 Oct 2005 15:55:02 -0000	1.10
+++ starmath/source/cfgitem.hxx	9 Nov 2005 15:04:54 -0000
@@ -38,7 +38,7 @@
 #ifndef _MATH_CFGITEM_HXX_
 #define _MATH_CFGITEM_HXX_
 
-#include <stl/vector>
+#include <vector>
 
 #include <com/sun/star/beans/PropertyValues.hpp>
 #include <com/sun/star/uno/Sequence.hxx>
Index: starmath/source/symbol.cxx
===================================================================
RCS file: /cvs/sw/starmath/source/symbol.cxx,v
retrieving revision 1.26
diff -u -r1.26 symbol.cxx
--- starmath/source/symbol.cxx	5 Oct 2005 15:05:59 -0000	1.26
+++ starmath/source/symbol.cxx	9 Nov 2005 15:05:01 -0000
@@ -35,7 +35,7 @@
 
 #pragma hdrstop
 
-#include <stl/vector>
+#include <vector>
 
 #ifndef _OSL_MUTEX_HXX_
 #include <osl/mutex.hxx>
cvs diff: Diffing uiconfig
cvs diff: Diffing uiconfig/smath
cvs diff: Diffing uiconfig/smath/accelerator
cvs diff: Diffing uiconfig/smath/accelerator/en-GB
cvs diff: Diffing uiconfig/smath/accelerator/en-US
cvs diff: Diffing uiconfig/smath/menubar
cvs diff: Diffing uiconfig/smath/statusbar
cvs diff: Diffing uiconfig/smath/toolbar
cvs diff: Diffing util
cvs diff: Diffing xml
